2022-10-08 14:55:15,081 (7f158c804808) :  INFO (core:349) - Starting framework core - Version: 2.6.3, Build: 819d367 (Fri Sep 23 01:15:17 UTC 2022)
2022-10-08 14:55:15,081 (7f158c804808) :  DEBUG (core:361) - Using the elevated policy
2022-10-08 14:55:15,082 (7f158c804808) :  DEBUG (core:365) - Extending whitelist: ['cgi', 'tempfile']
2022-10-08 14:55:15,082 (7f158c804808) :  DEBUG (core:450) - Starting runtime component.
2022-10-08 14:55:15,084 (7f158c804808) :  DEBUG (core:450) - Starting caching component.
2022-10-08 14:55:15,084 (7f158c804808) :  DEBUG (core:450) - Starting data component.
2022-10-08 14:55:15,084 (7f158c804808) :  DEBUG (core:450) - Starting networking component.
2022-10-08 14:55:15,085 (7f158c804808) :  DEBUG (networking:261) - Loaded HTTP cookies
2022-10-08 14:55:15,085 (7f158c804808) :  DEBUG (networking:428) - Setting the default network timeout to 20.0
2022-10-08 14:55:15,086 (7f158c804808) :  DEBUG (core:450) - Starting localization component.
2022-10-08 14:55:15,087 (7f158c804808) :  INFO (localization:409) - Setting the default locale to en-us
2022-10-08 14:55:15,087 (7f158c804808) :  WARNING (data:179) - Error decoding with simplejson, using demjson instead (this will cause a performance hit) - Expecting property name enclosed in double quotes: line 26 column 1 (char 1078)
2022-10-08 14:55:15,092 (7f158c804808) :  DEBUG (localization:427) - Loaded en strings
2022-10-08 14:55:15,092 (7f158c804808) :  DEBUG (core:450) - Starting messaging component.
2022-10-08 14:55:15,092 (7f158c804808) :  DEBUG (core:450) - Starting debugging component.
2022-10-08 14:55:15,092 (7f158c804808) :  DEBUG (core:450) - Starting services component.
2022-10-08 14:55:15,093 (7f158c804808) :  DEBUG (core:450) - Starting myplex component.
2022-10-08 14:55:15,093 (7f158c804808) :  DEBUG (core:450) - Starting notifications component.
2022-10-08 14:55:15,233 (7f158c804808) :  DEBUG (accessor:68) - Creating a new model access point for provider com.plexapp.system in namespace 'metadata'
2022-10-08 14:55:15,247 (7f15894aeb38) :  DEBUG (services:265) - Plug-in is not daemonized - loading services from system
2022-10-08 14:55:15,247 (7f158c804808) :  DEBUG (runtime:1117) - Created a thread named 'load_all_services'
2022-10-08 14:55:15,248 (7f15894aeb38) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/messaging/function/X0J1bmRsZVNlcnZpY2U6QWxsU2VydmljZXM_/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMApyMAo_'
2022-10-08 14:55:15,249 (7f158c804808) :  DEBUG (runtime:1117) - Created a thread named 'get_server_info'
2022-10-08 14:55:15,250 (7f158c804808) :  DEBUG (core:150) - Finished starting framework core
2022-10-08 14:55:15,250 (7f158c804808) :  DEBUG (core:560) - Loading plug-in code
2022-10-08 14:55:15,250 (7f1589428b38) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400'
2022-10-08 14:55:15,262 (7f15894aeb38) :  ERROR (networking:196) - Error opening URL 'http://127.0.0.1:32400/:/plugins/com.plexapp.system/messaging/function/X0J1bmRsZVNlcnZpY2U6QWxsU2VydmljZXM_/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoxCmRpY3QKMApyMAo_'
2022-10-08 14:55:15,266 (7f15894aeb38) :  ERROR (services:272) - Unable to load services from system. Loading from the current bundle only.
2022-10-08 14:55:15,271 (7f1589428b38) :  ERROR (networking:196) - Error opening URL 'http://127.0.0.1:32400'
2022-10-08 14:55:15,278 (7f158c804808) :  DEBUG (core:566) - Finished loading plug-in code
2022-10-08 14:55:15,279 (7f1589428b38) :  WARNING (core:541) - Unable to retrieve the machine identifier or server version.
2022-10-08 14:55:15,279 (7f15894aeb38) :  DEBUG (services:362) - Loaded services
2022-10-08 14:55:15,279 (7f158c804808) :  DEBUG (runtime:640) - Adding a prefix handler for 'System' ('/system')
2022-10-08 14:55:15,286 (7f158944bb38) :  DEBUG (services:438) - No shared code to load
2022-10-08 14:55:15,916 (7f158c804808) :  INFO (__init__:35) - Starting System 819d367 (Fri Sep 23 01:15:17 UTC 2022)
2022-10-08 14:55:15,916 (7f158c804808) :  DEBUG (runtime:640) - Adding a prefix handler for 'Player' ('/player')
2022-10-08 14:55:15,923 (7f158c804808) :  INFO (datakit:93) - Loaded the dictionary file
2022-10-08 14:55:15,923 (7f158c804808) :  DEBUG (agentservice:52) - Starting the agent service
2022-10-08 14:55:16,043 (7f158c804808) :  DEBUG (accessor:68) - Creating a new model access point for provider com.plexapp.system in namespace 'metadata'
2022-10-08 14:55:16,049 (7f158c804808) :  DEBUG (bundleservice:65) - Starting the bundle service
2022-10-08 14:55:16,050 (7f158c804808) :  DEBUG (runtime:1117) - Created a thread named 'start'
2022-10-08 14:55:16,050 (7f158c804808) :  DEBUG (playerservice:12) - Starting the player service
2022-10-08 14:55:16,050 (7f158c804808) :  DEBUG (flagservice:13) - Starting the media flag service
2022-10-08 14:55:16,051 (7f158c804808) :  DEBUG (runtime:1117) - Created a thread named 'start'
2022-10-08 14:55:16,051 (7f158c804808) :  DEBUG (peerservice:141) - Starting the peer service
2022-10-08 14:55:16,053 (7f1589265b38) :  DEBUG (runtime:1117) - Created a thread named 'ensure_agent_info_exists_inner'
2022-10-08 14:55:16,053 (7f158c804808) :  DEBUG (runtime:1117) - Created a thread named 'refresh_servers'
2022-10-08 14:55:16,053 (7f15891fcb38) :  DEBUG (peerservice:164) - Fetching the current list of servers
2022-10-08 14:55:16,054 (7f158c804808) :  DEBUG (runtime:1150) - Scheduled a timed thread named 'refresh_servers'
2022-10-08 14:55:16,054 (7f158c804808) :  DEBUG (streamservice:18) - Starting the stream service
2022-10-08 14:55:16,054 (7f158c804808) :  CRITICAL (runtime:709) - Private handlers are no longer supported; couldn't register <bound method StreamService.token_handler of <streamservice.StreamService object at 0x7f1588f71e10>>
2022-10-08 14:55:16,055 (7f15891fcb38) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/servers'
2022-10-08 14:55:16,056 (7f158c804808) :  DEBUG (runtime:1150) - Scheduled a timed thread named 'invalidation_timer'
2022-10-08 14:55:16,056 (7f1589265b38) :  DEBUG (runtime:1117) - Created a thread named 'ensure_agent_info_exists_inner'
2022-10-08 14:55:16,057 (7f158c804808) :  DEBUG (proxyservice:12) - Starting the proxy service
2022-10-08 14:55:16,057 (7f15891fcb38) :  ERROR (networking:196) - Error opening URL 'http://127.0.0.1:32400/servers'
2022-10-08 14:55:16,060 (7f158c804808) :  DEBUG (runtime:1117) - Created a thread named 'update_attribution_flags_inner'
2022-10-08 14:55:16,061 (7f158c804808) :  INFO (core:611) - Started plug-in
2022-10-08 14:55:16,061 (7f158c804808) :  DEBUG (socketinterface:160) - Starting socket server
2022-10-08 14:55:16,063 (7f158c804808) :  DEBUG (runtime:1117) - Created a thread named 'start'
2022-10-08 14:55:16,063 (7f1589265b38) :  DEBUG (runtime:1117) - Created a thread named 'ensure_agent_info_exists_inner'
2022-10-08 14:55:16,063 (7f158c804808) :  INFO (socketinterface:184) - Socket server started on port 36473
2022-10-08 14:55:16,064 (7f158c804808) :  INFO (pipeinterface:25) - Entering run loop
2022-10-08 14:55:16,064 (7f158c804808) :  DEBUG (runtime:717) - Handling request GET /:/prefixes
2022-10-08 14:55:16,067 (7f158c804808) :  DEBUG (runtime:814) - Found route matching /:/prefixes
2022-10-08 14:55:16,069 (7f15891fcb38) :  CRITICAL (runtime:1128) - Exception in thread named 'refresh_servers' (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-819d3678c/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/runtime.py", line 1126, in _start_thread
    f(*args, **kwargs)
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-819d3678c/System.bundle/Contents/Code/peerservice.py", line 169, in refresh_servers
    servers_el = self.get_servers_el()
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-819d3678c/System.bundle/Contents/Code/peerservice.py", line 165, in get_servers_el
    return XML.ElementFromURL('http://127.0.0.1:32400/servers', cacheTime = 0)
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-819d3678c/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/api/parsekit.py", line 344, in ElementFromURL
    method=method,
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-819d3678c/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/api/networkkit.py", line 67, in _http_request
    req = self._core.networking.http_request(url, *args, **kwargs)
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-819d3678c/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 346, in http_request
    return HTTPRequest(self._core, url, data, h, url_cache, encoding, errors, timeout, immediate, sleep, opener, follow_redirects, method)
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-819d3678c/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 118, in __init__
    self.load()
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-819d3678c/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 158, in load
    f = self._opener.open(req, timeout=self._timeout)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 435, in open
    response = meth(req, response)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 548, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 473, in error
    return self._call_chain(*args)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 407, in _call_chain
    result = func(*args)
  File "/usr/lib/plexmediaserver/Resources/Python/python27.zip/urllib2.py", line 556, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 503: Service Unavailable

2022-10-08 14:55:16,069 (7f1589265b38) :  DEBUG (runtime:1117) - Created a thread named 'ensure_agent_info_exists_inner'
2022-10-08 14:55:16,071 (7f158c804808) :  DEBUG (runtime:924) - Response: [200] MediaContainer, 486 bytes
